catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchFilterVisitor.java

```diff
@@ -135,21 +137,30 @@ public Object visit(DWithin filter, Object data) {
             // Example:
             // <ogc:Literal>org.geotools.geometry.jts.spatialschema.geometry.primitive.PointImpl@dc33f184</ogc:Literal>
             Literal literalWrapper = (Literal) filter.getExpression2();
+            Object geometryExpression = literalWrapper.getValue();
 
-            // Luckily we know what type the geometry expression should be, so
-            // we
-            // can cast it
-            PointImpl point = (PointImpl) literalWrapper.evaluate(null);
-            double[] coords = point.getCentroid()
-                    .getCoordinate();
             double distance = filter.getDistance();
 
-            LOGGER.debug("point: coords[0] = {},   coords[1] = {}", coords[0], coords[1]);
-            LOGGER.debug("radius = {}", distance);
-
-            openSearchFilterVisitorObject.setSpatialSearch(new SpatialDistanceFilter(coords[0],
-                    coords[1],
-                    distance));
+            if (geometryExpression instanceof PointImpl) {
+                PointImpl point = (PointImpl) literalWrapper.evaluate(null);
+                double[] coords = point.getCentroid()
+                        .getCoordinate();
+                LOGGER.debug("point: coords[0] = {},   coords[1] = {}", coords[0], coords[1]);
+                LOGGER.debug("radius = {}", distance);
```

Comment:
✏️ I would use `trace` here instead because it is expected behavior: https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels